src/common/libutil/Makefile.am

```diff
@@ -109,6 +110,12 @@ libutil_la_SOURCES = \
 	basename.c \
 	ansi_color.h
 
+if MAC
+libutil_la_LDFLAGS = -F/System/Library/Frameworks \
+									 -framework CoreServices \
+									 -framework SystemConfiguration
+endif
```

Comment:
I don't think the -F is needed since `/System/Library/Frameworks` is the default location?